### PR TITLE
trying to fix e2e

### PR DESCRIPTION
**Pull request by okwme**

### CODE REVIEW

File: tests/e2e/keys.go [hunk 1]
```diff
@@ -38,7 +37,7 @@ func createMemoryKey() (mnemonic string, info *keyring.Record, err error) {
 }
 
 func createMemoryKeyFromMnemonic(mnemonic string) (*keyring.Record, error) {
-	cdc := params.MakeTestEncodingConfig().Codec
+	// cdc := params.MakeTestEncodingConfig().Codec
```
Comment (okwme) [inline tests/e2e/keys.go]:
```suggestion
```

**Comment by okwme** [comment]
so the keys issue has been upstreamed here but gaia is still failing to produce blocks.
not sure how to debug the logs from the docker images though, @alexanderbez do you have an idea how to approach this?

**Comment by okwme** [comment]
merging this in to focus on the liveness tests as a separate PR

File: tests/e2e/validator.go [hunk 2]
```diff
@@ -129,9 +131,8 @@ func (v *validator) createConsensusKey() error {
 }
 
 func (v *validator) createKeyFromMnemonic(name, mnemonic string) error {
-	cdc := params.MakeTestEncodingConfig().Codec
```
Comment (yaruwangway) [inline tests/e2e/validator.go:132]: here is `cdc := gaia.MakeTestEncodingConfig().Codec`

Comment (yaruwangway) [reply tests/e2e/validator.go:132]: see here https://github.com/cosmos/gaia/pull/1483/files

File: tests/e2e/e2e_setup_test.go [hunk 3]
```diff
@@ -52,9 +51,9 @@ type IntegrationTestSuite struct {
 	valResources   map[string][]*dockertest.Resource
 }
 
-func TestIntegrationTestSuite(t *testing.T) {
-	suite.Run(t, new(IntegrationTestSuite))
-}
+//func TestIntegrationTestSuite(t *testing.T) {
```
Comment (yaruwangway) [inline tests/e2e/e2e_setup_test.go]: this is the entry point to test e2e, when i comment out, there is no test to run, if i only comment out ibc test, will also show no test to run. @okwme

File: tests/e2e/validator.go (hunk 2 above)
Comment (okwme) [reply tests/e2e/validator.go:132]: @alexanderbez noted that this should be removed in light of a globally available `cdc`. Unless was this the culprit of the keys encoding issue?

File: tests/e2e/e2e_setup_test.go (hunk 3 above)
Comment (okwme) [reply tests/e2e/e2e_setup_test.go]: are you suggesting i uncomment it or leave it as is?
i believe this change came from rebasing the `rho` branch with your changes, or?

Comment (yaruwangway) [reply tests/e2e/e2e_setup_test.go]: yeah, i should not comment out, if we still want to run e2e

File: tests/e2e/validator.go (hunk 2 above)
Comment (yaruwangway) [reply tests/e2e/validator.go:132]: hi @okwme , i think yours is right!

File: tests/e2e/e2e_setup_test.go (hunk 3 above)
Comment (glnro) [reply tests/e2e/e2e_setup_test.go]: Is this bc of IBC?

**Pull request merged**